tests/commands/contributors.py

```diff
@@ -6,56 +6,110 @@
 # or later license. See the LICENSE file for a copy of the license and the
 # AUTHORS file for copyright and authorship information.
 
+from collections import OrderedDict
+from dateutil.parser import parse as parse_datetime
+from email.utils import formataddr
+
 import pytest
 
 from django.core.management import call_command
+from django.core.management.base import CommandError
+
+from pootle.core.utils.timezone import make_aware
+from pootle_app.management.commands.contributors import Command
+from pytest_pootle.fixtures.contributors import CONTRIBUTORS_WITH_EMAIL
+
+
+def test_contributors_get_aware_datetime():
+    """Get an aware datetime from a valid string."""
+    instance = Command()
+
+    # Test ISO 8601 datetime.
```

Comment:
Is this really useful? Parsing is delegated to the underlying library, which is [already tested](https://github.com/dateutil/dateutil/blob/master/dateutil/test/test_parser.py). If something is not (properly) tested there or is missing, improving upstream's test suite sounds like the way to go to me.